### DISCUSSION TITLE
Has anyone gotten teleport to install on CentOS 7

### DISCUSSION THREAD

**Discussion by cshabazian** [gravitational/teleport#9046] [Q&A] [answered]
Has anyone gotten teleport to work on CentOS 7?  I've tried rpm, .tar file, and building from source, and can't seem to get it to connect (build fails, but others fail to connect).  If anyone has gotten it running and has some pointers, or a how-to, I'd appreciate it.

**Comment by webvictim** [comment, marked as answer]
Use the CentOS 6 binaries on https://goteleport.com/teleport/download/ - they're compiled against a lower glibc version and will also work on CentOS 7.

This is the link to download 8.0.0: https://get.gravitational.com/teleport-v8.0.0-linux-amd64-centos6-bin.tar.gz

**Comment by cshabazian** [comment]
Those compiled binaries seem to work, thanks.

**Comment by Xiol** [comment]
Unfortunately this means we can't use the repository at `rpm.releases.teleport.dev` for CentOS 7.

Is there any plan to support specific releases with that repository?

**Reply by gryzli-the-bugbear** [reply]
What I've ended doing is: 
1) Set the repo to disabled by default inside: /etc/yum.repos.d/teleport.repo
`enabled=0`

2) Then list the latest 7.x version (which is running okay under Centos 7 ) 
`yum --enablerepo teleport --showduplicates list teleport`

3) Finally install the version I want
`yum --enablerepo teleport install teleport-7.3.7-1`

Leaving the 'enabled=0' flag is an important one, or you risk auto-updating to the latest available version (which is 8 by now) and ruining your setup. 

I've also made a blog post about it: 
https://gryzli.info/2021/12/16/install-and-configure-teleport/
